conf/messages

```diff
@@ -25,6 +25,8 @@ caption = Transport details
 
 checkYourAnswers.heading = Check your answers
 checkYourAnswers.title = Check your answers - Transport details
+transportMeans.departure.heading = Check your answers
```

Comment:
Should the caption/subheading not be 'Transport details - Departure means of transport'

Reply:
should the key not be 
transportMeans.departure.checkYourAnswers.heading